Backend/DataAccessLayer/Repositories/OperationRepository.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DataAccessLayer.Models;
+using System.Data.Entity;
+using DataAccessLayer.Database;
+
+namespace DataAccessLayer.Repositories
+{
+    public class OperationRepository
+    {
+
+        public int createService(Service service, DatabaseContext _db)
+        {
+            try
+            {
+                _db.Services.Add(service);
+                return 1;
+
+            } 
+            catch(Exception)
+            {
+                return 0;
+            }
+        }
+
+        public int createService(string serviceName, DatabaseContext _db)
+        {
+            try
+            {
+                Service service = new Service
+                {
+                    ServiceName = serviceName,
+                    Disabled = false
+                };
+
+                _db.Services.Add(service);

```

Comment:
How about using the overloaded method `createService(Service service, DatabaseContext _db)` for this line?